students/saleta_sebastian/lesson_09_reading_and_writing_files/mad_libs.py

```diff
@@ -0,0 +1,21 @@
+def mad_libs(adjective, noun, verb, noun2):
```

Comment:
According to this task, you should have txt file that contain example text ```The ADJECTIVE panda: ....````. In the next step, you should open the file, read each line and check each word. In a case when a word is equal to one of the ADJECTIVE, NOUN, ADVERB, or VERB you should change this word to this passed by the user.

Reply:
I came up with solution re.sub(/bword/b, new_word) check this please in my new commit. If you want I can do this in this way that you write above.